vms/platformvm/warp/validator_test.go

```diff
@@ -304,3 +305,43 @@ func TestSumWeight(t *testing.T) {
 		})
 	}
 }
+
+func BenchmarkGetCanonicalValidatorSet(b *testing.B) {
+	pChainHeight := uint64(1)
+	subnetID := ids.GenerateTestID()
+	numNodes := 10_000
+	getValidatorOutputs := make([]*validators.GetValidatorOutput, 0, numNodes)
+	for i := 0; i < numNodes; i++ {
+		nodeID := ids.GenerateTestNodeID()
+		blsPrivateKey, err := bls.NewSecretKey()
+		if err != nil {
+			b.Fatal(err)
+		}
+		blsPublicKey := bls.PublicFromSecretKey(blsPrivateKey)
+		getValidatorOutputs = append(getValidatorOutputs, &validators.GetValidatorOutput{
+			NodeID:    nodeID,
+			PublicKey: blsPublicKey,
+			Weight:    20,
+		})
+	}
+
+	for _, size := range []int{0, 1, 10, 100, 1_000, 10_000} {
+		getValidatorsOutput := make(map[ids.NodeID]*validators.GetValidatorOutput)
+		for i := 0; i < size; i++ {
+			validator := getValidatorOutputs[i]
+			getValidatorsOutput[validator.NodeID] = validator
+		}
+		validatorState := &validators.TestState{
+			GetValidatorSetF: func(_ context.Context, _ uint64, _ ids.ID) (map[ids.NodeID]*validators.GetValidatorOutput, error) {
```

Comment:
```suggestion
			GetValidatorSetF: func(context.Context, uint64, ids.ID) (map[ids.NodeID]*validators.GetValidatorOutput, error) {
```
Because none of these args are used, we don't need the underscores.

Reply:
I like the underscores to make it explicit ignore, but will remove since omitting all arg names does the same.

Reply:
Updated